bazel/wasm/wasm.bzl

```diff
@@ -45,6 +45,9 @@ wasm_binary = rule(
 
 def wasm_cc_binary(name, **kwargs):
     wasm_name = "_wasm_" + name
+    kwargs.setdefault("additional_linker_inputs", ["@proxy_wasm_cpp_sdk//:jslib"])
+    kwargs.setdefault("linkopts", ["--js-library external/proxy_wasm_cpp_sdk/proxy_wasm_intrinsics.js"])
```

Comment:
Shouldn't this be:
```
kwargs.setdefault("linkopts", ["--js-library", "external/proxy_wasm_cpp_sdk/proxy_wasm_intrinsics.js"])
```
i.e. 2 parameters in the list?

Reply:
I am not sure why.  It works this way and it looks cleaner.  What is the advantage of adding the additional pair of quotes and comma?